panelapp/panels/tests/test_genepanels.py

```diff
@@ -176,9 +177,9 @@ def test_delete_gene(self):
             "number_of_genes"
         )  # 4 is due to create_batch
 
-        old_gps = GenePanel.objects.get(pk=gps.panel.pk).genepanelsnapshot_set.last()
-        assert old_gps.version != gps.version
-        assert old_gps.has_gene(gene_symbol) is True
+        old_gps = HistoricalSnapshot.objects.get(panel__pk=gps.panel.pk)
+        assert '{}.{}'.format(old_gps.major_version, old_gps.minor_version) != gps.version
```

Comment:
I know we have a mix of `assert ` and `self.assert*` calls - but we should really be using `self.assert*` as it has better support for error reporting and `assert` can be disabled if python is run with `optimise` param.
It's optional, so feel free to leave it for now, but it's something to improve over time.